herzog/parser.py

```diff
@@ -26,8 +28,8 @@ def __init__(self, cell_type: CellType, lines: Iterable[str]):
                 else:
                     self.lines.append(line)
             elif CellType.markdown == self.cell_type:
-                if '"""' == line:
-                    pass
+                if line.startswith('#'):
```

Comment:
Would this invalidate the following?
```
with herzog.Cell("markdown"): 
    """
    # cool header
    lorem ipsum doler
    """
```

Reply:
It would.

Reply:
@aofarrel We'd have to update the current notebooks.

Reply:
I also have an extra commit relevant to this that I haven't posted yet in case Brian's still reviewing.  Will add it when Brian gives the go ahead.

Reply:
Unless that extra commit makes it so the old syntax is still valid I think I will still be grumpy about this conversion. 

I'd be fine with both being an option, like how WDL accepts {two kinds} <<<of brackets>>> to delineate the command section, but not # becoming the new standard.